plugins/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ if (BUILD_PLUGIN_TRAJECTORY)
     add_subdirectory(trajectory)
 endif()
 
+if (BUILD_PLUGIN_STAC)
```

Comment:
Should this be a plugin or an always-on driver? The dependencies and vendored code are such that it could be always-on.